splinterd/sample_configs/splinterd.toml.example

```diff
@@ -128,3 +128,32 @@ oauth_client_secret = "<client-secret>"
 
 # Redirect URL for the OAuth provider used by the REST API
 oauth_redirect_url = "http://localhost:8080/oauth/callback"
+
+
+#Appenders is the structure and stdout is the items name
+[appenders.stdout]
+#Type of appender options are stdout,stderr,file,rolling_file
+kind = "stdout"
+#Pattern controls the formatting of each log message.
+#Full reference `https://docs.rs/log4rs/1.0.0/log4rs/encode/pattern/index.html`
```

Comment:
The relevant pieces of this documentation should be described in Splinter's documentation directly as it becomes part of Splinter's overall API (since config files are pieces of the API). Accordingly, this should be a URL to the relevant Splinter doc.

Reply:
Splinter docs don't have any references to the pattern struct currently, once log-config is stabilized the docs will generate/get pulled in. Should I include a follow up PR to link those docs?

Reply:
The docs are necessary, not as concerned about the link.